.github/workflows/acceptance.yml

```diff
@@ -100,8 +100,6 @@ jobs:
 
       - name: webpack
         env:
-          # this is fine to not have for forks, it shouldn't fail
-          SENTRY_WEBPACK_WEBHOOK_SECRET: ${{ secrets.SENTRY_WEBPACK_WEBHOOK_SECRET }}
```

Comment:
@asottile-sentry Would you mind removing this secret once the PR is merged? I cant access that part of the repo settings else I'd do it

Reply:
given the comment I can probably just remove it now right?

Reply:
Probably yes, but I dont know anything about this project and I dont really trust the comments :D